src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -1149,6 +1149,147 @@ RppStatus rppt_flip_voxel_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** remap ********************/
+
+RppStatus rppt_remap_host(RppPtr_t srcPtr,
+                          RpptDescPtr srcDescPtr,
+                          RppPtr_t dstPtr,
+                          RpptDescPtr dstDescPtr,
+                          Rpp32f *rowRemapTable,
+                          Rpp32f *colRemapTable,
+                          RpptDescPtr tableDescPtr,
+                          RpptInterpolationType interpolationType,
+                          RpptROIPtr roiTensorPtrSrc,
+                          RpptRoiType roiType,
+                          rppHandle_t rppHandle)
+{
+    RppLayoutParams layoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
+
+    if (interpolationType != RpptInterpolationType::NEAREST_NEIGHBOR && interpolationType != RpptInterpolationType::BILINEAR)
+        return RPP_ERROR_NOT_IMPLEMENTED;
```

Comment:
This check is present on the host, but not in hip in the API below

Reply:
Added it